dozer-integrations/dozer-spring-support/dozer-spring4/src/test/java/com/github/dozermapper/spring/DozerBeanMapperFactoryBeanTest.java

```diff
@@ -1,105 +1,104 @@
-/*

```

Comment:
has anything changed? if not, can you revert to make commit cleaner.

Reply:
Unused `import org.dozer.loader.api.BeanMappingBuilder;` was removed and together with move it seems has such effect.